scout/adapter/mongo/query.py

```diff
@@ -256,7 +258,9 @@ def build_query(
             if criterion in ["hgnc_symbols", "gene_panels"]:
                 gene_query = self.gene_filter(query, build=build)
                 if len(gene_query) > 0 or "hpo" in query.get("gene_panels", []):
-                    mongo_query["hgnc_ids"] = {"$in": gene_query}
+                    mongo_query["hgnc_ids"] = {
+                        EXCLUDE_CRITERION[bool(query.get("gene_panels_exclude"))]: gene_query
```

Comment:
Cool construction! The name of the dict isn't helping me quickly understand what it does. How about something like `GENE_PANELS_EXCLUDE_OPERATOR`?

Reply:
Sure!

Reply:
I'll try to clarify a bit more, but I wouldn't use **GENE_PANELS_**EXCLUDE_OPERATOR, because the operator is a general one that could be applied for other query parameters

Reply:
✅ Ah, yes, could be useful! `CRITERION_EXCLUDE_OPERATOR`?

Reply:
I wrote EXCLUDE_FROM_SEARCH_OPERATOR, do you like it or you prefer CRITERION_EXCLUDE_OPERATOR?

Reply:
Well, since you ask. 😉 Ideally we should stay clear of "exclude" as it is an either include or not include operation - a general set operator basically. Perhaps `SEARCH_SET_OPERATOR` or `CRITERION_SET_OPERATOR`?